appinventor/components/src/com/google/appinventor/components/runtime/util/JavaStringUtils.java

```diff
@@ -194,6 +195,14 @@ public static String joinStrings(List<Object> listOfStrings, String separator) {
     return join(listOfStrings, separator);
   }
 
+  public static List<String> split(String text, String at){
```

Comment:
To address the broken test, we should change the return type here to YailList and then call YailList.makeList(...) to wrap the output of this function as a YailList.